tests/selenium/test_get_next_results.py

```diff
@@ -0,0 +1,30 @@
+import time
+
+import pytest
+
+from pages.treeherder import Treeherder
+
+
+@pytest.fixture
+def sample_push(sample_data):
+    sample_push = []
+    for i in range(61):
+        sample_push.append({
+            'revision': 'revision{}'.format(i),
+            'push_timestamp': time.time() + (i * 3600),
+            'author': 'Test author <test@example.com>',
+            'revisions': [{
+                'comment': 'Test commit',
+                'author': 'Test author <test@example.com>',
+                'revision': 'revision{}'.format(i)
+            }]
+        })
+    return sample_push
+
+
+@pytest.mark.parametrize('count', ((10), (20), (50)))
+def test_get_next_results(base_url, selenium, count, push_stored):
```

Comment:
Should `push_stored` be `sample_push`?

Reply:
I need it to be stored though. I could overwrite the `push_stored` fixture myself, but this seemed cleaner.

Reply:
Oh you can override fixtures by declaring them locally in pytest, I didn't realise. Ah so `push_stored` uses the updated `sample_push`, makes sense.